Mathlib/Analysis/SpecialFunctions/Trigonometric/Inverse.lean

```diff
@@ -365,7 +366,7 @@ theorem arccos_le_pi_div_four {x} : arccos x ≤ π / 4 ↔ √2 / 2 ≤ x := by
     · intro
       linarith
 
-@[continuity]
+@[continuity, fun_prop]
 theorem continuous_arccos : Continuous arccos :=
```

Comment:
could you add the `ContinuousAt` version?

Reply:
`fun_prop` can use `Continuous` to prove `ContinuousAt`. I'll add the usual dot notation lemmas in another PR (added to my TODO list).